.doc_gen/metadata/cognito-identity-provider_metadata.yaml

```diff
@@ -282,6 +282,15 @@ cognito-identity-provider_AdminInitiateAuth:
   synopsis: start authentication with &COG; and administrator credentials.
```

Comment:
I see several code examples with snippets that are not in the metadata or in the ReadMe: ListUsers, ListUserPools, ConfirmDevice....shouldn't they be in the metadata and the ReadMe?

Reply:
Added to the metadata and readme where necessary.